client/src/App.js

```diff
@@ -47,168 +66,83 @@ class App extends React.Component {
+      //http://http://localhost:3004/dates${id}
+      //http://booking-dev2.us-west-1.elasticbeanstalk.com/booking/dates${id}
+       axios.post(`http://localhost:3004/booking/dates${id}`, {
           startDate: startDate,
           endDate: endDate,
-          adults: this.state.adults,
-          children: this.state.children,
-          infants: this.state.infants
+          guests: this.state.guests
         })
         .then(result => {
-  
           this.handleGetBookedDates()
-          
           this.setState({
-            startDate: new Date(),
-            endDate: new Date(),
-            message: result.data
+            successMessage: result.data.validDates
           })
-  
+
           setTimeout(() =>{
             this.setState({
-              message: ''
+              successMessage: ''
             })
           }, 3000)
-  
         })
         .catch(err => {
-          console.log(err)
-        })
-     }
-
-     handleChangeStart = (startDate) => {
-
-      startDate = startDate || this.state.startDate;
-  
-      let endDate = this.state.endDate;
-  
-      if(moment(startDate).isAfter(endDate)){
-        endDate = startDate
-        this.setState({endDate})
-      }
-  
-      this.setState({startDate})
-    
-    };
-
-
-    handleChangeEnd = (endDate) => {
-      endDate = endDate || this.state.endDate;
-      const startDate = this.state.startDate
-      if(moment(startDate).isAfter(endDate)){
-        endDate = startDate
-      }
-      this.setState({endDate})
-  
-    }
+          this.setState({
+            errorMessage: err.response.data.invalidDates
+          })
 
-  
+        setTimeout(() =>{
+          this.setState({
+            errorMessage: ''
+          })
+        }, 3000)
 
-    // Guest Logic
-    // Adults
-    increaseAdultCount = () => {
-      this.setState((prevState) => ({
-        adults: prevState.adults + 1
-      }))
-    }
-    
-    decreaseAdultCount = () => {
-      if(this.state.adults > 0){
-        this.setState((prevState) => ({
-          adults: prevState.adults - 1
-        }))
-      }
-    }
-  
-    // Children
-    increaseChildCount = () => {
-      this.setState((prevState) => ({
-        children: prevState.children + 1
-      }))
-    }
-  
-    decreaseChildCount = () => {
-      if(this.state.children > 0){
-        this.setState((prevState) => ({
-          children: prevState.children - 1
-        }))
-      }
-    }
-  
-    // Infants
-    increaseInfantCount = () => {
-      this.setState((prevState) => ({
-        infants: prevState.infants + 1
-      }))
-    }
-  
-    decreaseInfantCount = () => {
-      if(this.state.infants > 0){
-        this.setState((prevState) => ({
-          infants: prevState.infants - 1
-        }))
-      }
-    }
-  
+        })
+     }
 
     // Life Cycle Methods
     componentDidMount(){
       this.handleGetBookedDates()
     }
 
+  render(){
+
+    let successModal;
```

Comment:
Oh i missed this before on first review, but instead of doing an if/else, you can just set `sucessModel = null` here, that way you don't need your else statement